Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -478,6 +492,13 @@
       reagents:
       - ReagentId: TearGas
         Quantity: 20
+  - type: CartridgeAmmo #Sunrise-Start
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)